go/vt/vtgate/vindexes/region_json.go

```diff
@@ -32,12 +32,17 @@ import (
 	"vitess.io/vitess/go/vt/log"
 )
 
+const (
+	regionJSONParamRegionBytes = "region_bytes"
+	regionJSONParamRegionMap   = "region_map"
+)
```

Comment:
this is not referred in `rb, err := strconv.Atoi(m["region_bytes"])`

Reply:
whoops fixed